### PR TITLE
Added RIPEMD-160 to the configure file.

### PR DESCRIPTION
**Pull request by Bren2010**
Before, it was impossible to include RIPEMD-160 into your SJCL build because it wasn't included in the `./configure` file.

### CODE REVIEW

**Comment by Bren2010** [comment]
Also fixed some tests--RIPEMD-160 (and a couple of other things) weren't being tested with `make test`.

File: Makefile [hunk 1]
```diff
@@ -59,21 +59,23 @@ TEST_COMMON=  browserTest/nodeUtil.js test/test.js
 TEST_SCRIPTS= $(TEST_COMMON) \
               test/aes_vectors.js test/aes_test.js \
               test/bitArray_vectors.js test/bitArray_test.js \
-              test/ocb2_vectors.js test/ocb2_test.js  \
-              test/ccm_vectors.js test/ccm_test.js  \
+							test/bn_vectors.js test/bn_test.js \
```
Comment (Nilos) [inline Makefile]: wrong indentation!

File: test/ecc_conv.js [hunk 2]
```diff
@@ -4,15 +4,29 @@ new sjcl.test.TestCase("ECC convenience test", function (cb) {
     cb && cb();
     return;
   }
-  
-  try {  
+
+  try {
+    // Uses hard-coded randomness just to give the test something to work with.
+    // This isn't how this is supposed to be used in the real world.
+    var random = [ -625324409,
```
Comment (Nilos) [inline test/ecc_conv.js]: why?

**Comment by Bren2010** [comment]
@Nilos Sorry about the indentation!  The reason for the changes to `ecc_conv` is that when it's run in the *terminal*, you get the error:

> *** FAIL *** ECC convenience test: NOT READY: generator isn't seeded

because of something about the way tests are sandboxed.  I remember it working in the browser, though.

Adding the random array was the only way I could find to get the test to work in the terminal and the browser.

**Comment by Bren2010** [comment]
Can we please merge this?  Its holding up other projects.

**Comment by clark800** [comment]
:+1: I could really use this, any chance of an npm release soon?

**Comment by Nilos** [comment]
npm release: done!

**Pull request merged**